### DISCUSSION TITLE
--target=node6 equivalent for React SSR?

### DISCUSSION THREAD

**Discussion by ruballos** [parcel-bundler/parcel#7190] [General]
I need two compiles to achieve server-side rendering with React. I have spent many days trying and it does not work. With Parcel 1 it worked perfectly. My problem is that after compiling it tells me that Node does not recognize the css that it is going to send to the client. In parcel I just put --target = node6 and everything worked great. Help !!

Here is my actual package.json:

```json
"scripts": {
    "build:client": "cross-env BABEL_ENV=client parcel build src/index.html --target=brow --reporter @parcel/reporter-bundle-analyzer --no-source-maps",
    "build:server": "cross-env BABEL_ENV=server parcel build server/index.js --target=server --no-source-maps --log-level=info",
    "build": "rimraf dist && rimraf dist-server && npm run build:client && npm run build:server && node dist-server/",
    "start": "npm -s run build && node dist-server/index.js",
    "generate-build-version": "node generate-build-version",
    "postbuild": "npm run generate-build-version",
    "watch": "parcel serve src/index.html --lazy --open",
    "dev": "set PORT=3007 && react-scripts --max_old_space_size=7096 start",
    "format": "prettier --write \"src/**/*.{js,jsx}\"",
    "lint": "eslint \"src/**/*.{js,jsx}\" --quiet"
  },
  "targets": {
    "brow": {
      "publicUrl": "./dist/",
      "outputFormat": "commonjs",
      "distDir": "./dist",
      "context": "browser",
      "includeNodeModules": true
    },
    "server": {
      "context": "node",
      "distDir": "./dist-server",
      "engines": {"node": "8"},
      "outputFormat": "commonjs",
      "includeNodeModules": false
    }
  },
```

Here is the error:
SyntaxError: Unexpected token '.'
    at Object.compileFunction (node:vm:352:18)
dist-server/index.css:1
.line-total{border-top:2px solid #000}

**Comment by mischnic** [comment]
Please provide an full code sample

**Reply by ruballos** [reply]
Hi, here is my repo:
[](https://github.com/ruballos/parcelrssr)

I would appreciate your help infinitely

**Reply by ruballos** [reply]
https://github.com/ruballos/parcelrssr

**Reply by ruballos** [reply]
run "npm run build"

**Comment by devongovett** [comment]
Maybe use something like https://www.npmjs.com/package/ignore-styles?

**Reply by ruballos** [reply]
Hi, here is my repo:
[](https://github.com/ruballos/parcelrssr)

I would appreciate your help infinitely

**Reply by ruballos** [reply]
https://github.com/ruballos/parcelrssr

**Reply by ruballos** [reply]
run "npm run build"
